### PR TITLE
[DOCFIX] Update menu to 3 tiers

### PR DESCRIPTION
**Pull request by ccmao1130**
### What changes are proposed in this pull request?

Update main nav menu to 3 tiers
Rename Basic Logging to Logging & update respective link paths
Pull Glossary to level 1

### Why are the changes needed?

restructuring menu 

### Does this PR introduce any user facing changes?

webui

### CODE REVIEW

**Review by Zijian-Zhu — APPROVED** [review]

File: docs/en/Overview.md [hunk 1]
```diff
@@ -8,13 +8,17 @@ Welcome to Alluxio Documentation! You will find resources regarding deploying Al
 ## Get Started with Alluxio
 
 ### Install Alluxio Locally 
-Check out our [Quick Start Guide]({{ '/en/Get-Started.html' | relativize_url }}) for step-by-step guidance on how to download and configure Alluxio, run Alluxio on a local machine, and perform basic tasks via Alluxio Shell.
+Check out our [Quick Start Guide]({{ '/en/deploy/Get-Started.html' | relativize_url }}) for step-by-step guidance on how to download and configure Alluxio, run Alluxio on a local machine, and perform basic tasks via Alluxio Shell.
```
Comment (Xenorith) [inline docs/en/Overview.md:11]: remove

File: docs/en/deploy/Install-Alluxio-On-Docker.md [hunk 2]
```diff
@@ -0,0 +1,618 @@
...
+
+for line in textFile_RDD.collect():
+  print (line)
+```
+
+Congratulations, you've deployed a basic Dockerized Alluxio cluster! Read on to learn more about how to manage the cluster and make is production-ready.
+
+## Advanced Setup
+
+### Launch Alluxio with Java 11
```
Comment (Xenorith) [inline docs/en/deploy/Install-Alluxio-On-Docker.md:287]: check with @ssz1997 but i believe we build our images to launch using java 11 by default. if true, then this section should be removed

File: docs/en/deploy/Install-Alluxio-On-Docker.md [hunk 3]
```diff
@@ -0,0 +1,618 @@
...
+Starting from v2.9.0, Alluxio processes can be launched with Java 11 inside Docker containers by pulling the `alluxio/alluxio-jdk11` image from Dockerhub.
+
+To use java 11 image, replace `alluxio/alluxio` with `alluxio/alluxio-jdk11` in the command launching Alluxio Docker container.
+
+### Launch Alluxio with the development image
+
+Starting from v2.6.2, a new docker image, `alluxio-dev`, is available in Dockerhub for development usage. Unlike the default `alluxio/alluxio` image that
```
Comment (Xenorith) [inline docs/en/deploy/Install-Alluxio-On-Docker.md:294]: can we remove all the "starting from version x" where x is a 2.x version

File: docs/en/deploy/Install-Alluxio-On-Docker.md [hunk 4]
```diff
@@ -0,0 +1,618 @@
...
+       -Dalluxio.master.hostname=localhost" \
+    alluxio/{{site.ALLUXIO_DOCKER_IMAGE}} proxy
+```
+
+See [Properties List]({{ '/en/reference/Properties-List.html' | relativize_url }}) for more
+configuration options for Alluxio proxy server.
+
+## Performance Optimization
+
+### Enable short-circuit reads and writes
```
Comment (Xenorith) [inline docs/en/deploy/Install-Alluxio-On-Docker.md:520]: this feature is gone in 3.x

**Review by Xenorith — CHANGES_REQUESTED** [review]

File: docs/en/Overview.md (hunk 1 above)
Comment (ccmao1130) [reply docs/en/Overview.md:11]: i moved the file back to the deploy folder

File: docs/en/operation/Logging.md [hunk 5]
```diff
@@ -32,7 +32,11 @@ Each user will have separate log files.
 ### Server Logs
 
 Log files for each individual Alluxio server process (e.g., master, worker, FUSE, proxy) can be found under `${ALLUXIO_HOME}/logs/`.
+<<<<<<< HEAD
+They are named as `master.log`, `master.out`, `worker.log`, `worker.out`, `job_master.log`, `job_master.out`, 
+=======
 Each process should have two corresponding files ending with `.log` or `.out`, such as `worker.log` and `worker.out` for the worker process.
+>>>>>>> d6e3ea14c4ea1e488202c6f1982daf3c739e669b
```
Comment (Xenorith) [inline docs/en/operation/Logging.md:39]: uh oh

File: docs/_data/menu-en.yml [hunk 6]
```diff
@@ -1,175 +1,180 @@
...
 
-  - title: Core Services
-    buttonTitle: Core_Services
-    subfiles:
-      - title: Data Caching
-        url: /en/core-services/Data-Caching.html
-      - title: Metadata Cache & Invalidation
-        url: /en/core-services/Metadata-Caching.html
+- title: What is Alluxio
+  buttonTitle: What_Is_Alluxio
```
[the remark this answers is not in the document]
Comment (ccmao1130) [reply docs/_data/menu-en.yml:10]: no..........

**Review by Xenorith — APPROVED** [review]

**Comment by Xenorith** [comment]
```
2023-08-17T20:25:36.3588182Z + /usr/src/alluxio/dev/scripts/src/alluxio.org/checkdocs
2023-08-17T20:25:36.4235902Z 2023/08/17 20:25:36 error matching content of menu.yml with acutally list of docs in directory of docs/ with message : 
2023-08-17T20:25:36.4236501Z  error matching menu.yml with list of docs in directory of docs, following docs are not in the menu.yml: [/en/compute/Tensorflow.md]
```

**Comment by Xenorith** [comment]
alluxio-bot, merge this please

**Pull request merged**